guides/doc-Project_API/topics/proc_using-the-get-http-method.adoc

```diff
@@ -0,0 +1,34 @@
+[id="using-the-get-http-method"]
+= Using the GET HTTP method
+
+Use the GET HTTP verb to get data from the API about an existing entry or resource.
```

Comment:
same as above; I'd use "method" in favor of "verb". (not blocking your PR)

Reply:
```suggestion
Use the GET HTTP method to get data from the API about an existing entry or resource.
```